includes/class-newspack-newsletters.php

```diff
@@ -277,18 +277,15 @@ public static function rest_api_init() {
 		);
 		\register_rest_route(
 			'newspack-newsletters/v1',
-			'mailchimp/(?P<id>[\a-z]+)/interest/(?P<interest_id>[\a-z]+)',
+			'mailchimp/(?P<id>[\a-z]+)/segments',
 			[
 				'methods'             => \WP_REST_Server::EDITABLE,
-				'callback'            => [ __CLASS__, 'api_set_mailchimp_interest' ],
+				'callback'            => [ __CLASS__, 'api_set_mailchimp_segments' ],
 				'permission_callback' => [ __CLASS__, 'api_authoring_permissions_check' ],
 				'args'                => [
```

Comment:
`tag_ids` should be listed as well.

Reply:
Fixed in 90d99c86765f6afd243ede7f5c4d9be397aec354